ex/auto/writer/odev_build_form/build_form.py

```diff
@@ -124,111 +135,89 @@ def create_form(self, doc: XTextDocument) -> None:
             name="text-1",
             label="show only sales since",
             comp_kind=Forms.CompenentKind.FixedText,
-            x=2,
+            x=x,
             y=80,
             width=35,
             height=6,
         )
 
         #  radio buttons inside a group box; use a property change listener
+        col2_x = 103
```

Comment:
Same thing here on `x`. Just redefine again `x = 103` unless there is a good reason no to: e.g: current value of `x` is still needed. This makes much easier code to read.

Reply:
Let's resolve in first comment about this issue.